chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -692,6 +692,11 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             .webchat__basic-transcript__activity-markdown-body > :first-child {
                 margin-top: 0px;
             }
+            .webchat__basic-transcript__activity-markdown-body img.webchat__render-markdown__external-link-icon {
```

Comment:
Is this the default styling on WebChat?

Would it make more sense to add a class name to only override the external link icon for system message such as `.ms_lcw_webchat_system_message`?

This way, it'd make sure that we only override the link inside the system message and not else where 

Reply:
To make sure that we only override the link inside the system message I have referred to webchat__basic-transcript__activity-markdown-body class. In constants.ts file we have defined for system message we are using this class.

<img width="691" alt="image" src="https://github.com/microsoft/omnichannel-chat-widget/assets/128818969/d07a216c-9eb0-45e1-afc5-dd4287ec695f">

Shall I change webchat__basic-transcript__activity-markdown-body to .ms_lcw_webchat_system_message? or it is ok to keep the original class name as it is.